libdevcore/CommonData.h

```diff
@@ -394,4 +393,23 @@ std::vector<T> make_vector(Args&&... _args)
 	return result;
 }
 
+// Function that check if all characters in string are safe and return quoted parametr.
+// If there is unsafe characters it use "0x" + h256(_value, h256::AlignLeft).hex();
+std::string formatAsStringOrNumber(std::string const &_value)
+{
+	std::string resultString = "\"";
+	for (int i = 0; i < _value.length; i++)
+	{
+		if (!isalpha(_value[i]))
+		{
+			return "0x" + h256(_value, h256::AlignLeft).hex();
+		}
+		else
+		{
+			resultString += _value[i];
```

Comment:
No need to copy each character individually, you can just return the full string (prefix and postfixed with quotes) at the ond.